test/api/sse-dashboard.test.js

```diff
@@ -0,0 +1,313 @@
+jest.mock("../../server_src/config/auth");
+
+const EventSource = require("eventsource");
+const { parse } = require("flatted/cjs");
+const dbHandler = require("../db-handler");
+const supertest = require("supertest");
+const getEndpoints = require("express-list-endpoints");
+const { setupTestApp } = require("../../app-test");
+
+let request;
+const routeBase = "/dashboardInfo/";
+const ssePath = "get/";
+
+beforeAll(async () => {
+  await dbHandler.connect();
+  const server = await setupTestApp();
+
+  const endpoints = getEndpoints(server);
+  expect(endpoints).toContainEqual({
+    methods: ["GET"],
+    middleware: ["anonymous", "anonymous"],
+    path: routeBase + ssePath
+  });
+  request = supertest(server);
+});
+
+describe("SSE-dashboard", () => {
+  it("should be able to be called with an EventSource", async (done) => {
+    const getRequest = request.get(routeBase + ssePath);
+    const url = getRequest.url;
+    expect(url).toBeTruthy();
+
+    const es = new EventSource(url);
+
+    // events.emit('test', 'test message')
+    es.onmessage = (e) => {
+      const parsedMsg = parse(e.data);
+
+      expect(parsedMsg).toMatchObject({
+        printerInformation: expect.any(Array), // TODO printerInformation instead of printersInformation?
+        currentOperations: {
+          operations: expect.any(Array), // TODO printerInformation instead of printersInformation?
+          count: {
+            printerCount: expect.any(Number),
```

Comment:
This is the place I applied the `toMatchObject` and `expect.any` functions to validate both value (f.e. null, as well as type mixed together)